roles/kafka_broker/tasks/register_cluster.yml

```diff
@@ -32,8 +42,45 @@
       ]
     status_code: 204
   run_once: true
+  when: auth_mode != 'mtls'
   no_log: "{{mask_secrets|bool}}"
   register: output
   until: output.status == 204
   retries: "{{ mds_retries }}"
   delay: 10
+
+- name: Register Kafka Cluster mTLS
+  uri:
+    url: "{{mds_bootstrap_server_urls.split(',')[0]}}/security/1.0/registry/clusters"
+    method: POST
+    validate_certs: false
+    headers:
+      Content-Type: application/json
+      Authorization: "Bearer {{ authorization_token }}"
```

Comment:
why do we have authorization token in mtls only ?

Reply:
It got added as I copied the above task and created this one. 
Although i dont think it makes any difference if we pass the token or not. If we pass it the principal will be taken from the token. and token was given for kafka broker cert. And if dont pass the token principal will be taken from the cert of kafka broker. so same principal will be used either way.

Reply:
agreed, it would pass the same token and does not make any difference on the output, but since this is misleading, I think we should remove the token field

Reply:
cool